packages/syft/src/syft/service/api/api.py

```diff
@@ -508,9 +529,48 @@ def exec_code(
         code: PrivateAPIEndpoint | PublicAPIEndpoint,
         context: AuthedServiceContext,
         *args: Any,
+        log_id: UID | None = None,
         **kwargs: Any,
     ) -> Any:
+        # stdlib
+        import builtins as __builtin__
```

Comment:
nit: print override probably better in a standalone function if we can to avoid doing too much in exec_code

Reply:
Fixed! by : [a0c91fb](https://github.com/OpenMined/PySyft/pull/9199/commits/a0c91fbfa025cf8e2cf67e8d8d28d00de8b446ab)